config/dev.exs

```diff
@@ -28,6 +28,8 @@ config :movie_wager_api, MovieWagerApi.Endpoint,
 # Do not include metadata nor timestamps in development logs
 config :logger, :console, format: "[$level] $message\n"
 
+config :movie_wager_api, :client, url: "http://localhost:4200"
```

Comment:
Not sure I follow why this shouldn't be in source. It's not a secret that the app runs at localhost:4200 in development. Why do you feel this should be hidden?